### PR TITLE
docs: improve lang()

### PR DESCRIPTION
**Pull request by kenjis**
**Description**
- improve descriptions

**Checklist:**
- [x] Securely signed commits
- [] Component(s) with PHPDoc blocks, only if necessary or adds value
- [] Unit testing, with >80% coverage
- [x] User guide updated
- [] Conforms to style guide

### CODE REVIEW

File: user_guide_src/source/general/common_functions.rst [hunk 1]
```diff
@@ -106,13 +106,15 @@ Service Accessors
 
 .. php:function:: lang($line[, $args[, $locale]])
 
-    :param string $line: The line of text to retrieve
+    :param string $line: The language filename and the key of the text to retrieve.
     :param array  $args: An array of data to substitute for placeholders.
-    :param string $locale: Specify a different locale to be used instead of default one.
+    :param string $locale: Specify a different locale to be used instead of the current locale.
+    :returns: The text in the language file
+    :rtype: list<string>|string
```
Comment (datamweb) [inline user_guide_src/source/general/common_functions.rst:113]: @kenjis , is that(`rtype`) a typo?

Comment (datamweb) [reply user_guide_src/source/general/common_functions.rst:113]: However, I didn't see anything in [documentation](https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/documentation.rst) or [sphinx](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html/). Please provide a reference link.

Comment (kenjis) [reply user_guide_src/source/general/common_functions.rst:113]: No. It is for `Return type:`. See https://github.com/search?q=repo%3Abcit-ci%2FCodeIgniter%20:rtype:&type=code

**Review by datamweb — APPROVED** [review]

File: user_guide_src/source/general/common_functions.rst (hunk 1 above)
Comment (michalsn) [reply user_guide_src/source/general/common_functions.rst:113]: `rtype` is documented in python domain: https://www.sphinx-doc.org/en/master/usage/domains/python.html#info-field-lists

Comment (datamweb) [reply user_guide_src/source/general/common_functions.rst:113]: Thank you @michalsn .

**Pull request merged**